backend/uclapi/dashboard/templates/getStarted.html

```diff
@@ -20,7 +19,11 @@
     window.staticURL = "{% static "" %}";
   </script>
   <div class="app"></div>
-  <script src="{% static "js/getStarted.js" %}"></script>
+  {% if WEBPACK_SERVE %}
```

Comment:
Is there a less hacky solution to this? Perhaps set the static path to http://localhost:8080 using an environment variable?

Reply:
that's what I was doing but webpack-serve (http://localhost:8080) doesn't serve images and for that we had to go back to the Django static file server

Reply:
Ahhh I see. Is this an inbuilt restriction in Webpack? It seems strange that it wouldn't serve the files. Does this explain it at all? https://stackoverflow.com/questions/34870888/serving-static-assets-in-webpack-dev-server

Full disclosure: I know literally nothing about Webpack and very little about frontend so there might be something strange about our setup that prevents this from happening.

Reply:
I did think of that, but I wasn't sure how it'll affect the serving of images as now the entire images folder will now be moved to `frontend/`. It's a lot of work to implement it as I'll need to change the code for images on the frontend, but I'll still give it a try.